tools/gulp/docs/generateDocPage.js

```diff
@@ -16,6 +16,11 @@ function generateDocPage(routes, page, docsPath, rootPath) {
       return '';
     }
 
+    // On the client-side the "root" variable is defined via Webpack,
+    // but we also need to define it here for "server-side" rendering
+    process.env = Object.assign(process.env, {
```

Comment:
Can this be just `process.env.root = rootPath`?

Reply:
Derp, yup. Fixed